up-l1/README.adoc

```diff
@@ -271,11 +271,13 @@ The following table are the valid combinations, which maps to the corresponding
 |===
 | source `resource_id` | sink `resource_id` | Publish | Notification | Request | Response | Use Case
 
-| [8000-FFFE] |   None   |    V    |              |         |         | A uE listens for a specific published message 
+| [8000-FFFE] |   None   |    V    |              |         |         | A uE listens for a specific published message
 | [8000-FFFE] |     0    |         |      V       |         |         | A uE listens for a specific notification message
-|      0      | [1-7FFF] |         |              |    V    |         | A uE listens for a specific request 
+|      0      | [1-7FFF] |         |              |    V    |         | A uE listens for a specific request
 |   [1-7FFF]  |     0    |         |              |         |    V    | A uE listens for a specific response
-|     FFFF    |     0    |         |      V       |         |    V    | A uE listens for all notifications and responses 
+|     FFFF    |   None   |    V    |              |         |         | A uE listens for all published messages
```

Comment:
Don't think this is valid to listen to all published messages, that would mean everything is sent to the uE

Reply:
So we don't need this in uStreamer case? If not, I will remove this.
Tag @PLeVasseur for more comments

Reply:
Not for publish no.

Reply:
Hi @evshary -- sorry I missed this. It's true. We do not listen for all Publish messages.

We have to use the uSubscription's notification of which topics should be forwarded out / listened for in order to select them one-by-one by a specific topic UUri.

This is because there is no destination, so if we listened for (and forwarded) all Publish messages, we could find all devices getting flooded by messages they did have an subscribing uEs on.

Reply:
Thanks for both of your feedback. I removed it.